staging/src/k8s.io/kubectl/pkg/cmd/explain/explain.go

```diff
@@ -172,16 +171,49 @@ func (o *ExplainOptions) Run() error {
 		}
 	}
 
+	// Fallback to openapiv2 implementation using special template name
 	if o.EnableOpenAPIV3 {
-		return explainv2.PrintModelDescription(
-			fieldsPath,
-			o.Out,
-			o.DiscoveryClient.OpenAPIV3(),
-			fullySpecifiedGVR,
-			recursive,
-			o.OutputFormat,
-		)
+		format := o.OutputFormat
+
+		if format == plaintextTemplateName {
```

Comment:
WDYT of making this into a switch statement, maybe with a fallback if v3 isn't supported?

Reply:
changed to switch stmt